changelog.md

```diff
@@ -4,6 +4,12 @@
 
 _Nothing yet..._
 
+## 2018-01-04 - Fixed bar hovering for bar chart component
+
+**Fixed:**
+- bpk-component-barchart: 2.1.27 => 2.1.28
```

Comment:
As a contributor you don't need to put the version numbers in, the person who is releasing the change will do this. Please revert.

Reply:
🤔  so just to check: you mean to leave out only `: 2.1.27 => 2.1.28`?

Reply:
Yeah just have: 
```
- bpk-component-barchart:
```